### ISSUE TITLE
Can this model run with multiple GPUs?

### ISSUE THREAD

**Issue by duythucvnu** [roboflow/rf-detr#35]
I'm training the model on a T4 GPU with 16GB VRAM, but I have to reduce the batch size to 2. Is it possible to run the model on multiple GPUs to handle larger batch sizes or improve performance?

**Comment by isaacrob-roboflow** [comment]
Hi! See issue https://github.com/roboflow/rf-detr/issues/17 for multi GPU training. 

If you only have the one GPU, you can use gradient accumulate steps to bring up your batch size. We use layernorm instead of batchnorm which makes training using gradient accumulate with a small per-step batch size stable.
